x-pack/plugins/security_solution/public/timelines/components/timeline/data_providers/providers.tsx

```diff
@@ -302,6 +308,7 @@ export const DataProvidersGroupItem = React.memo<DataProvidersGroupItem>(
                 toggleEnabledProvider={handleToggleEnabledProvider}
                 toggleExcludedProvider={handleToggleExcludedProvider}
                 toggleTypeProvider={handleToggleTypeProvider}
+                displayValue={displayValue}
                 val={getDataProviderValue(dataProvider)}
```

Comment:
consider always using `dataProvider.queryMatch.value` here (similar to https://github.com/elastic/kibana/pull/142436#discussion_r986210366)

Reply:
changed